lfs.c

```diff
@@ -24,6 +26,15 @@ static inline void lfs_cache_zero(lfs_t *lfs, lfs_cache_t *pcache) {
     pcache->block = LFS_BLOCK_NULL;
 }
 
+/// Invalidate the lookahead buffer. This is done during mounting and failed traversals ///
+static inline void lfs_setup_invalid_lookahead_buffer(lfs_t *lfs)
```

Comment:
Minor nits:
- Could this be named something like `lfs_alloc_reset` so the alloc functions (lfs_alloc, lfs_alloc_ack) all have a similar prefix?
- Could this be moved next to lfs_alloc_ack, it seems related. If you ordered the function definitions lfs_alloc_ack, lfs_alloc_reset, lfs_alloc, you wouldn't need to predeclare lfs_alloc_ack.
- Also I don't think inline is beneficial here, since it's static the compiler is very likely to inline if it does save code size

Reply:
I'm okay with all of these comments! Definitely appreciate consistency :)